mitiq/utils.py

```diff
@@ -0,0 +1,74 @@
+"""Utility functions."""
+
+import random
+
+import cirq
+
+
+def random_circuit(depth: int, **kwargs) -> cirq.Circuit:
```

Comment:
Do we need to use **kwargs here? Using seed=None would make the method's type signature more explicit to users.

Reply:
Made `seed` an optional arg.